.github/workflows/fern/fern-check.yml

```diff
@@ -0,0 +1,19 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - main
+      
+jobs:
+  fern-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Install Fern
+        run:  npm install -g fern-api
+
+      - name: Check Fern API is valid
+        run: fern check
```

Comment:
what does this check do? is it validation check or "fern is installed properly" check. If it's document validation I would move it to https://github.com/imartinez/privateGPT/blob/main/.github/workflows/tests.yml and add it as a mandatory condition

Reply:
It validates the OpenAPI spec and the configuration files within `/fern`

Reply:
More details: https://docs.buildwithfern.com/overview/cli/fern-check